ooniprobe/Utility/BackgroundTask.m

```diff
@@ -77,7 +77,9 @@ + (void)checkIn {
     [testSuite setStoreDB:NO];
     AbstractTest *test = [[AbstractTest alloc] initTest:testName];
     [test setStoreDB:NO];
+    [test setAutoRUn:YES];
     [test setAnnotation:YES];
+    test.settings.annotations[@"origin"] = @"autorun";
```

Comment:
I see you're already doing this in `AbstractTest.m`. Do we need to do it twice or can we avoid doing it here?